provider/k8s/k8s.go

```diff
@@ -258,9 +273,32 @@ func (p *Provider) initializeTemplates() error {
 		return errors.WithStack(err)
 	}
 
+	d, _ := p.Cluster.AppsV1().Deployments("cert-manager").Get("cert-manager", am.GetOptions{})
+
+	// println(d.Spec.Template.Labels["app.kubernetes.io/version"])
+
 	if p.CertManager {
-		if err := p.applySystemTemplate("cert-manager", nil); err != nil {
-			return errors.WithStack(err)
+		if d != nil && d.Spec.Template.Labels["app.kubernetes.io/version"] != "v1.9.0" {
```

Comment:
if we re-apply the manifest on top of existing resources, the operation will hang forever so only apply it if running a different cert-manager version